dashboard/management/commands/course.py

```diff
@@ -40,7 +40,7 @@ def handle(self, *args, **options):
             self.stdout.write ("Creating course view options")
             course_view_obj = CourseViewOption(
                 course_id=prefixed_course_id,
-                show_files_accessed=True,
+                show_resources_accessed=True,
```

Comment:
can you commit the rename of the `show_files_accessed` to `show_resources_accessed` master also?

Reply:
See #649